src/BlazorUI/Bit.BlazorUI.Extras/Components/RichTextEditor/@types/module.d.ts

```
@@ -0,0 +1,6 @@
+declare abstract class Module<T extends {} = {}> {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)